posthog/hogql_queries/query_date_range.py

```diff
@@ -0,0 +1,89 @@
+from datetime import datetime
+from functools import cached_property
+from typing import Optional
+
+import pytz
+from dateutil.relativedelta import relativedelta
+
+from posthog.models.team import Team
+from posthog.queries.util import get_earliest_timestamp
+from posthog.schema import DateRange, IntervalType
+from posthog.utils import DEFAULT_DATE_FROM_DAYS, relative_date_parse, relative_date_parse_with_delta_mapping
+
+
+# Originally copied from posthog/queries/query_date_range.py with some changes to support the new format
+class QueryDateRange:
+    """Translation of the raw `date_from` and `date_to` filter values to datetimes."""
+
+    _team: Team
+    _date_range: Optional[DateRange]
+    _interval: Optional[IntervalType]
+    _now_nontz: datetime
```

Comment:
Nit: `nontz` sounds like the german word for a moustache (though it isn't). Took me 1.5 seconds to realize it means non-timezone 😅.

I've learned it pays to be explicit with variable names, so perhaps `now_without_timezone` is cleaner? 🤷

Reply:
I changed it :)